packages/rum-core/test/common/apm-server.spec.js

```diff
@@ -179,6 +184,26 @@ describe('ApmServer', function() {
     }, 300)
   })
 
+  it('should log errors from apm-server', done => {
+    const apmServer = new ApmServer(configService, loggingService)
+    apmServer
+      .sendTransactions([
+        {
+          id: '21312',
+          span_count: 0,
+          duration: 100,
+          type: 'app'
+        }
+      ])
+      .catch(err => {
+        const trimmedMessage = err.message.replace(/\/n/g, '').trim()
+        expect(trimmedMessage).toEqual(
+          'http://localhost:8001/intake/v2/rum/events HTTP status: 400 error validating JSON document against schema: I[#] S[#] doesn\'t validate with "transaction#"\n  I[#] S[#/allOf/1] allOf failed\n    I[#] S[#/allOf/1/required] missing properties: "trace_id"'
```

Comment:
we can also remove the  new line indentation and print in single line. WDYT @jahtalab ?

Reply:
I don't have a strong opinion! I think if it is readable in the console, then there's no need to change the APM server response.

Reply:
cool, Actually its pretty neat with indentation in console. So ll keep it as it is.